infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -116,24 +121,30 @@ private void openTables() {
         }
     }
 
-    public Set<String> getStreamsToReplicate() {
+    /**
+     * This method is for ACTIVE to fetch streams to replicate from LOG_REPLICATION_STREAMS_INFO_TABLE.
```

Comment:
Streams to replicate must be initialized on standby as well on startup.  Consider the case where the standby upgrades and active is still in the old version.  The right behavior is that the standby has the new streams to replicate in the new version.

Reply:
Yeah I see your point, and that's also where I'm stuck. Because the point of this PR is to remove the static file of streams to replicate, and rely on the `is_federated` flag instead (by querying registry table in flight). In this case, STANDBY cannot know the full list of streams to replicate in advance, and it cannot differentiate the case between: 
1. A table sent from ACTIVE but not opened in STANDBY (such that STANDBY didn't find its `is_federated` flag in registry table) 
2. A table sent from ACTIVE is not expected because of rolling upgrade.

I haven't seen a feasible solution for it, I thought we might need to discuss offline...

Reply:
hmm... we can probably rely on the version field in the metadata

Reply:
I think on this direction we will fall into similar situation. Let's say a stream sent to STANDBY and it cannot find this stream in its RegistryTable, but found `is_upgraded` flag set, then we still cannot differentiate between:

- This stream is "not on STANDBY" -> rolling upgrade case
- This stream is "on STANDBY" but hasn't been opened